membership-attribute-service/app/prodtest/VariablesUpdatedOnSchedule.scala

```diff
@@ -0,0 +1,33 @@
+package prodtest
+
+import akka.actor.ActorSystem
+import com.gu.memsub.util.ScheduledTask
+import com.typesafe.scalalogging.LazyLogging
+import services.ScanamoFeatureToggleService
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+
+class VariablesUpdatedOnSchedule(featureToggleService: ScanamoFeatureToggleService)(implicit ec: ExecutionContext, system: ActorSystem) extends LazyLogging {
+
+  private var percentageTrafficForZuoraLookup = 0
```

Comment:
that's a roundabout way of saying that there's usually another way to do it that doesn't involve mutable state.  But better to do your own reading about var in scala to make your own mind up!